lttng-tools.te

```diff
@@ -28,7 +28,8 @@ allow lttng_sessiond_t self:capability2 block_suspend;
 allow lttng_sessiond_t self:process { setrlimit signal_perms };
 allow lttng_sessiond_t self:fifo_file rw_fifo_file_perms;
 allow lttng_sessiond_t self:tcp_socket listen;
-allow lttng_sessiond_t self:unix_stream_socket create_stream_socket_perms;
+allow lttng_sessiond_t self:unix_stream_socket { create_stream_socket_perms connectto };
+allow lttng_sessiond_t lttng_sessiond_tmpfs_t:file map;
```

Comment:
This line should go between line 40 and 41.

Reply:
Done.